### PR TITLE
add bic model

### PR DESCRIPTION
**Pull request by ylilan**
add bic model in Util module and execute that in fitting_sequence.py

### CODE REVIEW

**Comment by ylilan** [comment]
add bic to lenstronomy

File: lenstronomy/Util/analysis_util.py [hunk 1]
```diff
@@ -125,4 +125,16 @@ def ellipticities(I_xy, x, y):
...
     e2 = 2 * Q_xy / norm
-    return e1 / (1+bkg), e2 / (1+bkg)
\ No newline at end of file
+    return e1 / (1+bkg), e2 / (1+bkg)
+
+
+
+def bic_model(likelihood,num_data,num_param):
+    """
+    :param likelihood: logL, likelihood value
```
Comment (sibirrer) [inline lenstronomy/Util/analysis_util.py]: this is the log likelihood value, not the likelihood

File: lenstronomy/Workflow/fitting_sequence.py [hunk 2]
```diff
@@ -117,6 +117,24 @@ def best_fit_likelihood(self):
...
+        """
+        returns the bayesian information criterion of the model.
+        :return: bic value, float
+        """
+        num_data = self.likelihoodModule.num_data
+        num_param_nonlinear = self.param_class.num_param()[0]
+        num_param_linear = self.param_class.num_param_linear()
+        num_param = num_param_nonlinear + num_param_linear
+        bic = analysis_util.bic_model(self.best_fit_likelihood,num_data,num_param)
+        print "number of the free model parameters:", num_param
```
Comment (sibirrer) [inline lenstronomy/Workflow/fitting_sequence.py]: do you need the print statements in here? The function should "just" provide you with a specific output.

Comment (sibirrer) [reply lenstronomy/Workflow/fitting_sequence.py]: also: print() with bracket statements to be compatible with python3

File: test/test_Util/test_analysis_util.py [hunk 3]
```diff
@@ -45,6 +45,12 @@ def test_half_light_radius(self):
         r_half = analysis_util.half_light_radius(lens_light, x_grid, y_grid, center_x=0, center_y=0)
         assert r_half == -1
 
+    def test_bic_model(self):
+        bic=analysis_util.bic_model(0,np.e,1)
+        assert bic == 1
+
+
+
```
Comment (sibirrer) [inline test/test_Util/test_analysis_util.py]: please also test the bic() definition of the FittingSequence

**Pull request merged**